CanvasCore/CanvasCore/Helm/HelmSplitViewController.swift

```diff
@@ -97,6 +97,9 @@ extension HelmSplitViewController: UISplitViewControllerDelegate {
             top?.navigationItem.leftItemsSupplementBackButton = true
             if top?.isKind(of: EmptyViewController.self) == false {
                 top?.navigationItem.leftBarButtonItem = prettyDisplayModeButtonItem(displayMode)
+                if let observer = top as? SplitViewControllerObserverProtocol {
```

Comment:
Where this will fall down is with embedded view controllers.. eg if the `HorizontalMenuViewController` gets embedded in some other view controller (like a `ModuleItemDetailViewController` might seem plausible). Instead of a protocol, maybe send a notification via `NotificationCenter`?